shards/core/runtime.hpp

```diff
@@ -238,6 +238,8 @@ struct DefaultHelpText {
 
   static inline const SHOptionalString InputHelpAnyType = SHCCSTR("Input of any type is accepted.");
 
+  static inline const SHOptionalString InputHelpAnyButType = SHCCSTR("Input of any type is accepted. However, when comparing different types(e.g., boolean with integer), the shard will not throw an error, but will sometimes return unexpected results.");
```

Comment:
not sure on `unexpected`, sounds a bit "bad" ? haha
They are more "implementation detail" results, not sure how to define it.

Reply:
lexicographical results